src/TextureCubeMap.cpp

```diff
@@ -167,6 +195,10 @@ TextureCubeMap::State::CreateTexture() {
 
 void
 TextureCubeMap::State::DestroyTexture() {
+  if (externalTexture) {
+    // Texture life cycle is handled outside of this class (e.g TimeWarp Cubemap Layer)
```

Comment:
Should externalTexture be set to `0` here?

Reply:
I don't think it's needed, because a destroy/init would create a new texture. 

Updated the PR to set 0 on the normal texture handle.